src/Reader/readpar.f90

```diff
@@ -1329,6 +1338,22 @@ SUBROUTINE readdr(IO, EQN, DISC, BND, IC)
              DISC%DynRup%RS_iniSlipRate1 = RS_iniSlipRate1! V_ini1, initial sliding velocity
              DISC%DynRup%RS_iniSlipRate2 = RS_iniSlipRate2! V_ini2, initial sliding velocity
              DISC%DynRup%t_0      = t_0       ! forced rupture decay time
+             DISC%DynRup%ThermalPress = thermalPress !switches TP on (1) or off(0)
+             IF (DISC%DynRup%ThermalPress.EQ.1) THEN !additional parameters
+                 logInfo0(*) 'Thermal pressurization assumed'
+                 !pyhsical
+                 DISC%DynRup%TP_hwid = TP_hwid
+                 DISC%DynRup%alpha_th = alpha_th
+                 DISC%DynRup%alpha_hy = alpha_hy
+                 DISC%DynRup%rho_c = rho_c
+                 DISC%DynRup%TP_lambda = TP_lambda
+                 EQN%Temp_0 = IniTemp
```

Comment:
Why not using a consistent name in the namelist and the structure?
Also don't you want to use easi for initialisation of these parameters?

Reply:
easi is only well suited for parameters that (potentially) vary in space. Otherwise it's more overhead than useful.
What are these parameters and do they potentially vary?

Reply:
At the beginning TP parameters should not vary (just to make life easier).
There will be an updated version when everything is working where some parameters can vary.